ci/taos/menu.html

```diff
@@ -44,6 +44,7 @@
             <option value="/" selected>[ PR: details ]</option>
             <option value="./webhook.php">Webhook Handler</option>
             <option value="../gcov_html/">Code Coverage</option>
+            <option value="../taos/webapp/unittest_result/">Unit test</option>
```

Comment:
I think that the menu is NNStreamer specific feature. So I am not sure that we have to keep it in TAOS-CI repository.